target_chains/ton/contracts/tests/utils/wormhole.ts

```diff
@@ -0,0 +1,159 @@
+];
+
+export const GUARDIAN_SET_2 = [
+  "0x58CC3AE5C097B213CE3C81979E1B9F9570746AA5",
+  "0xFF6CB952589BDE862C25EF4392132FB9D4A42157",
+  "0x114DE8460193BDF3A2FCF81F86A09765F4762FD1",
+  "0x107A0086B32D7A0977926A205131D8731D39CBEB",
+  "0x8C82B2FD82FAED2711D59AF0F2499D16E726F6B2",
+  "0x11B39756C042441BE6D8650B69B54EBE715E2343",
+  "0x54CE5B4D348FB74B958E8966E2EC3DBD4958A7CD",
+  "0x66B9590E1C41E0B226937BF9217D1D67FD4E91F5",
+  "0x74A3BF913953D695260D88BC1AA25A4EEE363EF0",
+  "0x000AC0076727B35FBEA2DAC28FEE5CCB0FEA768E",
+  "0xAF45CED136B9D9E24903464AE889F5C8A723FC14",
+  "0xF93124B7C738843CBB89E864C862C38CDDCCCF95",
+  "0xD2CC37A4DC036A8D232B48F62CDD4731412F4890",
+  "0xDA798F6896A3331F64B48C12D1D57FD9CBE70811",
+  "0x71AA1BE1D36CAFE3867910F99C09E347899C19C3",
+  "0x8192B6E7387CCD768277C17DAB1B7A5027C0B3CF",
+  "0x178E21AD2E77AE06711549CFBB1F9C7A9D8096E8",
+  "0x5E1487F35515D02A92753504A8D75471B9F49EDB",
+  "0x6FBEBC898F403E4773E95FEB15E80C9A99C8348D",
+];
+
+export const GUARDIAN_SET_3 = [
+  "0x58CC3AE5C097B213CE3C81979E1B9F9570746AA5",
+  "0xFF6CB952589BDE862C25EF4392132FB9D4A42157",
+  "0x114DE8460193BDF3A2FCF81F86A09765F4762FD1",
+  "0x107A0086B32D7A0977926A205131D8731D39CBEB",
+  "0x8C82B2FD82FAED2711D59AF0F2499D16E726F6B2",
+  "0x11B39756C042441BE6D8650B69B54EBE715E2343",
+  "0x54CE5B4D348FB74B958E8966E2EC3DBD4958A7CD",
+  "0x15E7CAF07C4E3DC8E7C469F92C8CD88FB8005A20",
+  "0x74A3BF913953D695260D88BC1AA25A4EEE363EF0",
+  "0x000AC0076727B35FBEA2DAC28FEE5CCB0FEA768E",
+  "0xAF45CED136B9D9E24903464AE889F5C8A723FC14",
+  "0xF93124B7C738843CBB89E864C862C38CDDCCCF95",
+  "0xD2CC37A4DC036A8D232B48F62CDD4731412F4890",
+  "0xDA798F6896A3331F64B48C12D1D57FD9CBE70811",
+  "0x71AA1BE1D36CAFE3867910F99C09E347899C19C3",
+  "0x8192B6E7387CCD768277C17DAB1B7A5027C0B3CF",
+  "0x178E21AD2E77AE06711549CFBB1F9C7A9D8096E8",
+  "0x5E1487F35515D02A92753504A8D75471B9F49EDB",
+  "0x6FBEBC898F403E4773E95FEB15E80C9A99C8348D",
+];
+
+export const GUARDIAN_SET_4 = [
+  "0x5893B5A76C3F739645648885BDCCC06CD70A3CD3",
+  "0xFF6CB952589BDE862C25EF4392132FB9D4A42157",
+  "0x114DE8460193BDF3A2FCF81F86A09765F4762FD1",
+  "0x107A0086B32D7A0977926A205131D8731D39CBEB",
+  "0x8C82B2FD82FAED2711D59AF0F2499D16E726F6B2",
+  "0x11B39756C042441BE6D8650B69B54EBE715E2343",
+  "0x54CE5B4D348FB74B958E8966E2EC3DBD4958A7CD",
+  "0x15E7CAF07C4E3DC8E7C469F92C8CD88FB8005A20",
+  "0x74A3BF913953D695260D88BC1AA25A4EEE363EF0",
+  "0x000AC0076727B35FBEA2DAC28FEE5CCB0FEA768E",
+  "0xAF45CED136B9D9E24903464AE889F5C8A723FC14",
+  "0xF93124B7C738843CBB89E864C862C38CDDCCCF95",
+  "0xD2CC37A4DC036A8D232B48F62CDD4731412F4890",
+  "0xDA798F6896A3331F64B48C12D1D57FD9CBE70811",
+  "0x71AA1BE1D36CAFE3867910F99C09E347899C19C3",
+  "0x8192B6E7387CCD768277C17DAB1B7A5027C0B3CF",
+  "0x178E21AD2E77AE06711549CFBB1F9C7A9D8096E8",
+  "0x5E1487F35515D02A92753504A8D75471B9F49EDB",
+  "0x6FBEBC898F403E4773E95FEB15E80C9A99C8348D",
+];
+
+export function createGuardianSetUpgradeBytes(
```

Comment:
isn't there any function in wormhole sdk to do it? we might have some ourselves too.